.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "semi": false,
```

Comment:
Personally, I'm a fan of semicolons because it increases readability

Do you have an argument why you want to disable semis?